lib/hooks/event-wrapper.js

```diff
@@ -0,0 +1,5 @@
+class EventWrapper {
+
```

Comment:
This will become the thing that is passed to hook functions as an argument - it'll get helper methods to do useful things with a case, and will contain "useful" metadata relating to the underlying event.